libs/sdk-ui-kit/src/Header/generateHeaderMenuItemsGroups.ts

```diff
@@ -7,20 +7,31 @@ import { isFreemiumEdition, shouldHidePPExperience } from "../utils/featureFlags
  * @internal
  */
 export function generateHeaderMenuItemsGroups(
+    targetApp: "kpiDashboards" | "analyticalDesigner",
```

Comment:
This is not good. If you need to mention AD or KD, then this code should probably be in AD or KD, right?

Reply:
Also, why is `loadCsv` not there too?

Reply:
What this is supposed to do is specify, which items should be always available no matter what, right? Why is this different from other items? Can it happen so that you are in AD for example even though you don't have the menu item?

Reply:
The reason why I have not added `loadCsv` is that I am not sure what is the status on that; whether, we plan to have load in the header or not.

Reply:
And do you need the target app? See my last comment?

Reply:
Looking at it, I would guess that If certain FF are turned off such as `canCreateVisualization`, It would not make sense to have an AD tab in the header.